lib/matplotlib/tests/test_font_manager.py

```diff
@@ -317,7 +317,7 @@ def test_get_font_names():
             font = ft2font.FT2Font(path)
             prop = ttfFontProperty(font)
             ttf_fonts.append(prop.name)
-        except:
+        except:  # noqa: E722
```

Comment:
in the alternative, could be more specific about the exception or do `except Exception:` (skipping KeyboardInterrupt and SystemExit)

I did not dig into which exception(s) would be thrown here.

Reply:
I think it would be good to make the correct fix here if possible since that is what the purpose of adding these linter checks is :) Hopefully, it wouldn't be too hard to get the exception thrown your way. Or is this a case of it only fails on some remote systems? Worth a shot to fix properly I think.

Reply:
The only direct exception is a NotImplementedError raised in `ttfFontProperty()`. However, there are likely various possible internal exceptions.

I suggest to `except Exception:` here, because:
- we certainly don't want to intercept KeyboardInterrupt and SystemExit
- the whole try-except block has the purpose to recover and move on if anything goes wrong for a given font path. We certainly don't want to blow up just because some font has problems. Even if we knew the exceptions, I think we would still have to catch them all.

Reply:
Updated accordingly